### DISCUSSION TITLE
The purpose of the Issues and Discussions sections

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#2374] [General]
When I came first time here I noticed that the **Issues** section was enormously large and contained, in addition to real issues, questions, ideas and many others. At that time I think there were over 350 open issues. 

A few months ago I received the status of Collaborator which allowed me to get access to a higher level in the project and contribute concretely if I had done so until then. Creating the Discussions sections by Daniel was a small step but huge for all of us. The **Issues** section went through a complex triage and we kept only the real issues in OpenMage, that need immediate resolution. All others have been moved to the Discussions in different sections (request for help, ideas, extensions, ...).

**What do Issues and Discussions mean in my opinion?**

**ISSUES SECTION** - an issue identified in OpenMage that affects the correct functionality and requires immediate resolution.

**What happens when someone reports an issue?** 
OpenMage offers the possibility to add a label. As a Collaborator, I check the report content and edit it to be extremely clear and concise. Then I change the initial label after reading the post and add other labels. As a rule, their number should be as small as possible. I left only four:

**needs investigation** = the opinion of one or more people is requested to determine if it is an issue or not. if it is not the post is closed or moved to **Discussions** sections.

**waiting feedback** = there are already discussions but no conclusion has been reached.

**bug confirmed** = one or more people have confirmed the existence of the issue. at this moment a remedy is needed as quickly as possible.

**PR requested** = someone must implement the remedy. once implemented, the open issue is linked to the PR and it will close automatically after merging. at this stage only "bug confirmed" label should be visible.

Periodically I check the Issue section and choose a topic based on the labels. Obviously, where I see "PR requested" or "bug confirmed", I no longer choose them because I have confirmation of the issue. The other two remain visible, I start to analyze a problem maybe I can provide an opinion.

Maybe in the future I will rename the labels for simplicity **Investigation, Discussions, Bug, PR**.

**DISCUSSIONS SECTIONS** - anything unrelated to the first one. If I have an idea (no matter it is a feature request, code removing, discussing a change or a lack of clarity about an implementation), I post it in Discussions before . Maybe I want to make an improvement to the code, I don't post it in Issues but submit it here to the debate based on which we can create a PR.

**Comment by sreichel** [comment]
@addison74 turning a lot of issues into discussions was a big mistake - i'd like to revert.

Better change or add labels to the opened issue. Add a priority-system, ...
